docs/api/source/index.rst

```diff
@@ -16,17 +16,21 @@ a container.
 .. admonition:: Supported Protocols
    :class: caution
 
-   This library currently only supports an `early version of WebSocket
+   This library currently only supports an Inrupt `Enterprise Solid Server
+   <https://docs.inrupt.com/ess/latest/>`__ version of `WebSocket
    notifications
-   <https://docs.inrupt.com/ess/1.1/services/service-websocket/>`__, which
-   formed the basis of the Solid Notification Protocol 1.0 draft specification,
-   which is undergoing technical review as it matures into a formal
-   recommendation. A future release of this library will support this draft
-   specification once server implementations become available. 
+   <https://docs.inrupt.com/ess/1.1/services/service-websocket/>`__.
+   
+   The ESS Websocket Notifications protocol formed the basis of the new `Solid
+   Notification Protocol 1.0 <https://solid.github.io/notifications/protocol>`__
+   draft specification, which is undergoing technical review as it matures into
+   a formal recommendation.
```

Comment:
So ... in terms of wording, I wonder if we should be more "The ESS WebSocket Notifications is based on the draft Solid Notification Protocol. Features based on draft specifications are subject to change."

cc: @nicolasmondada - not sure how the Product team wants to present the product 😄

Reply:
Nope, that's not the case, the ESS WebSocket Notifications came first, as far as I understand it from @acoburn

Reply:
Oh ... sorry - I meant more that "it's in-line" but was sloppy in choosing a one-word alternative to in line and too lazy to look up in google.  Basically, I felt the more important point is that we're spec compliant or whatnot.

Reply:
Although, we're not with that particular version, I think.  Hence ... the wackadoodle wording, and my passing it off to Nick M 😄

Reply:
I think it's simpler to just consider it it's own thing. When we add support for a new spec it'll likely be a semver major change, I think (not wishing to expand on why right now)

Reply:
Just to clarify - sorry, my mind's all aflutter waiting for the delivery of a desk (and now its delay) while working on three other things  - but by "consider it its own thing", does that mean that we should remove the paragraph?  I'm okay with that, since I think(?) we're not in sync with the draft 1.0 version (although mostly we are?). As such, letting people know that it's the basis of the draft spec or the status of the spec as it undergoes the process may not provide meaningful-enough info for people?